pull_request_template.md

```diff
@@ -1,35 +1,42 @@
 # Description
 
-Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. Explain both the expected functionality and the edge cases. 
-
-## Type of change
+## Summary of the change / which issue is fixed / motivation / context:
 
+## Type of change:
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix a feature that knowingly causes existing functionality to not work as expected)
 
-# Test and Reproduce
-Outline the steps to test or reproduce the PR here.
+## Reproduce:
+Outline the steps required to see the intended effect of your PR:
+
+## Related code:
+- Which code (which methods/actions/components) could this change interact with? Please trace dependencies for every method/action/component that was modified:
 
-1. 
+- Who would be the best person to review each one of these code parts?
```

Comment:
This should evaluated by the reviewer, using common sense.
You should only approve if you feel you have the knowledge to do it.

Reply:
Agreed. But author can also put some thought into who to request specifically. While most people have some knowledge about most parts of the code, it could be useful to think about who knows the most about its ins and outs, previous gotchas, reasons for specific optimizations, etc